### ISSUE TITLE
Is impossible to connect many hydro generators in parallel?

### ISSUE THREAD

**Issue by shackra** [minetest-technic/technic#255]
After asking for help on [Reddit](https://www.reddit.com/r/Minetest/comments/3ts4oo/technic_how_in_the_world_do_you_setup_a/) regarding the setup of hydro generators, I'm facing another issue: it seems impossible to connect many generators to a same network:

![captura_22-11-2015-132006](https://cloud.githubusercontent.com/assets/1055216/11325767/0d010198-911c-11e5-96ad-a64c19f44a63.png)

As shown on the screenshot, I was able to connect two generators, but not the other four. am I'm doing something wrong or this is a technical impossibility of the mod?

**Comment by asl97** [comment]
iirc, graphic glitch happen all the time, try putting in water and check the amount of power produce.

**Comment by shackra** [comment]
ouh..........

That explains a lot! Thanks for the reply, I'll be careful the next time!
